content/cloud-computing/concepts/cloud-services/cloud-services.md

```diff
@@ -0,0 +1,68 @@
+---
+Title: 'Cloud Services'
+Description: 'There are many different cloud-computing services, but the big three are Microsoft Azure, Amazon Web Services (AWS) and Google Cloud Platform (GCP).'
+Subjects:
+  - 'Computer Science'
+Tags:
+  - 'Cloud Computing'
+  - 'Networking'
+CatalogContent:
+  - 'learn/foundations-of-cloud-computing'
+  - 'paths/back-end-engineer-career-path'
+---
+
+There are many different cloud-computing services, but the big three are [Amazon Web Services (AWS)](https://aws.amazon.com/), [Microsoft Azure](https://azure.microsoft.com/), and [Google Cloud Platform (GCP)](https://cloud.google.com/). All three offer services to cover the most significant cloud-computing needs:
+
+- [Infrastructure as a service (IaaS)](https://www.codecademy.com/resources/docs/cloud-computing/iaas)
+- [Platform as a service (PaaS)](https://www.codecademy.com/resources/docs/cloud-computing/paas)
+- [Function as a service (FaaS)](https://www.codecademy.com/resources/docs/cloud-computing/faas)
+- [Software as a service (SaaS)](https://www.codecademy.com/resources/docs/cloud-computing/saas)
+
+## AWS
+
+AWS is the largest cloud computing platform, and the most mature, introducing its first public cloud services in 2006. It serves millions of users, from individuals to large enterprises, to governments. Offering more than 200 services to its customers, AWS has the largest catalog of services of the big three. As of 2021 it has 25 geographic regions and 218+ edge locations.
```

Comment:
Do we have information anywhere on regions, availability zones, and edge locations?